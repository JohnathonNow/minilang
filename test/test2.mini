var X := 100

print('X = {X}\n')

X := old + 10

print('X = {X}\n')
