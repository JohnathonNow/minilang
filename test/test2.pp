Some random include file.
Blah blah blah.