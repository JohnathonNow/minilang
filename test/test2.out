X = 100
X = 110
